### ISSUE TITLE
Learninglocker + StatementsFilter related_acitivites raises an error

### ISSUE THREAD

**Issue by srad** [xabbuh/xapi#6]
As far as I can see the Learninglocker is pretty much a standard server, however, when I add the filter to load the related activities the Learninglocker server raises an error, that a flag is wrongly serialized:

``` php
    public function enableRelatedActivityFilter()
    {
        $this->filter['related_activities'] = true;
    }
```

must be changed to:

``` php
    public function enableRelatedActivityFilter()
    {
        $this->filter['related_activities'] = 'true'; // <--------------------- string
    }
```

Then it works. I don't know who violates which standard, but I had to overwrite your code.

**Comment by xabbuh** [comment]
Thank you for reporting this issue @srad. I'll look into it later tonight and will keep you informed whether or not this can be fixed on our side.

**Comment by xabbuh** [comment]
This is indeed an issue in the `StatementsFilter` class and will be fixed by php-xapi/model#11.

**Comment by xabbuh** [comment]
I released version `0.3.0` of the `php-xapi/model` package which fixes this issue.
